apps/dsp-app/cypress/support/e2e.ts

```diff
@@ -1,5 +1,15 @@
-import './commands/login';
 import { UserProfiles } from '../models/user-profiles';
+import './commands/data-model-class-command';
+import './commands/login';
+import './commands/ontology-command';
+
+const resizeObserverLoopErrRe = /^[^(ResizeObserver loop limit exceeded)]/;
```

Comment:
what is this for?

Reply:
I tried to get rid of those uncaught exceptions which flooding the console. Copied it from somewhere.